build/pipelines/pgo.yml

```diff
@@ -1,5 +1,27 @@
 trigger: none
 pr: none
+schedules:
+  - cron: "0 5 * * 2-6" # Run at 05:00 UTC Tuesday through Saturday (Even later than Localization, after the work day in Pacific, Mon-Fri)

```

Comment:
I've switched the PGO build to have its own built-in nightly schedule, rather than relying on the triggers from the Azure DevOps side.